.example.env

```diff
@@ -0,0 +1,7 @@
+# Separate Firebase project for local development
+FIREBASE_API_KEY="AIzaSyCBkQHeikIsiYZ2yOHiqH_mGJKDWMDU500"
```

Comment:
are the quotes required?

Reply:
Nope they seem to work without them, but it's [safer to use double-quotes](https://stackoverflow.com/a/33318886/5179469).